azure-pipelines.release.yml

```diff
@@ -0,0 +1,109 @@
+pr: none
+trigger: none
+
+variables:
+  - group: 'Github and NPM secrets'
+
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '10.x'
+    displayName: 'Install Node.js'
+
+  - script: |
+      git config user.name "UI Fabric Build"
+      git config user.email "fabrictactical@service.microsoft.com"
+      git remote set-url origin https://$(githubUser):$(githubPAT)@github.com/OfficeDev/office-ui-fabric-react.git
+    displayName: Authenticate git for pushes
+
+  - script: |
+      node common/scripts/install-run-rush.js install --bypass-policy
+    displayName: rush install
+
+  - script: |
+      npm run generate-version-files
+    displayName: npm run generate-version-files
+
+  - script: |
+      node common/scripts/install-run-rush.js build --production --verbose
+    displayName: rush build (Create production build)
+
+  - script: |
+      mkdir -p $(Build.ArtifactStagingDirectory)/api
```

Comment:
Not sure this will pass exit codes through properly, so you might want to either fix the exit code part if that's possible, or set this step to fail on stderr

Reply:
adding `&&` at the end.